src/AFQMC/Hamiltonians/RealDenseHamiltonian.cpp

```diff
@@ -0,0 +1,264 @@
+#include<cstdlib>
```

Comment:
Has the bug for distributed Hamiltonians been fixed?

Reply:
not yet